### DISCUSSION TITLE
Pressing Skip + Left/Right Arrow keys teleports the catcher to the Leftmost/Rightmost areas

### DISCUSSION THREAD

**Discussion by Nekomosh004** [ppy/osu#19696] [Ideas] [answered]
Just a minor thing I noticed. 

Pressing the skip key (Default: Space) **and** the Move left and right keys (Default: Left and Right Arrow keys) teleports the catcher to the Leftmost/Rightmost corner of the playing field.

Not an expected behavior Stable/CE does. 

Video:

https://user-images.githubusercontent.com/82471957/184018495-c685a07a-a705-458a-a823-cf325e3ccaaa.mp4

**Comment by smoogipoo** [comment, marked as answer]
https://github.com/ppy/osu/issues/9526
